templates/partials/candidate/financial-summary.html

```diff
@@ -58,18 +50,14 @@ <h3 class="t-ruled--bottom">Campaign committees</h3>
           <h4 class="callout__title">
             <a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}">{{ committee.name }}</a>
           </h4>
-          <span class="callout__subtitle">Principal campaign committee</span>
+          <span class="callout__subtitle term" data-term="principal campaign committee">Principal campaign committee</span>
         </div>
         <div class="callout__action">
-          <span class="t-block t-data">View detailed data:</span>
           <ul class="callout__sublinks">
-            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}&tab=receipts">Money received &raquo;</a></li>
-            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}&tab=disbursements">Money spent &raquo;</a></li>
-            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}&tab=filings">Reports filed &raquo;</a></li>
+            <li><a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle) }}&tab=receipts">Money received</a></li>
```

Comment:
I know this didn't change here, but now that I'm seeing these lines in the diff, I'm wondering why we have the `tab` parameter outside the `url_for` call. I think this would be a little easier to read and do the same thing like this:

```
<a href="{{ url_for('committee_page', c_id=committee.committee_id, cycle=cycle, tab='receipts') }}">Money received</a>
```